packages/notification-center/src/i18n/languages/gu.ts

```diff
@@ -1,10 +1,10 @@
 import { ITranslationEntry } from '../lang';
 
-export const GJ: ITranslationEntry = {
+export const GU: ITranslationEntry = {
   translations: {
     notifications: 'સૂચના',
     markAllAsRead: 'બધાને વાંચેલા તરીકે ચિહ્નિત કરો',
     poweredBy: 'દ્વારા સંચાલિત',
```

Comment:
Hello @ainouzgali ,
Greetings for the day, sorry but i didn't understand why it's conflicting
Pls help

Reply:
Hi @codingis4noobs2 :) Sorry, I meant merging from main and then fixing the conflict in lang.ts file. It happened because more languages were added in the meantime, not your fault😅 We are trying to figure out a better way, sorry for the inconvenience